rust-runtime/aws-smithy-runtime-api/src/client/runtime_components.rs

```diff
@@ -377,12 +377,37 @@ declare_runtime_components! {
     }
 }
 
+impl From<RuntimeComponents> for RuntimeComponentsBuilder {
+    fn from(rc: RuntimeComponents) -> Self {
+        RuntimeComponentsBuilder {
+            builder_name: "converted from RuntimeComponents",
```

Comment:
Perhaps we should have a separate `from_components` function that takes a builder name rather than implementing the `From` trait.

Reply:
Updated in 5a4e3b2.